packages/grafana-ui/src/components/Table/TableCell.tsx

```diff
@@ -32,15 +41,33 @@ export const TableCell: FC<Props> = ({ cell, field, tableStyles, onCellFilterAdd
     innerWidth -= tableStyles.lastChildExtraPadding;
   }
 
-  return (
-    <>
-      {cell.render('Cell', {
-        field,
-        tableStyles,
-        onCellFilterAdded,
-        cellProps,
-        innerWidth,
-      })}
-    </>
+  const link: LinkModel<any> | undefined = field.getLinks?.({
+    valueRowIndex: rowIndex,
+  })[0];
+
+  let onClick: MouseEventHandler<HTMLAnchorElement> | undefined;
+  if (link && link.onClick) {
```

Comment:
nit: can be simplified
```suggestion
  if (link?.onClick) {
```